src/reducers/GraphDataState.ts

```diff
@@ -25,7 +25,7 @@ export const INITIAL_GRAPH_STATE: GraphState = {
     showMissingSidecars: true,
     showNodeLabels: true,
     showSecurity: false,
-    showServiceNodes: false,
```

Comment:
We usually stick these defaults in `src/config.ts` and have `GraphDataState.ts` initialize from there. This way, all of the default settings are changed in one place. And in the future, we could also have overriding settings for dev, test, state, prod...

Reply:
Ok separate Jira is probably better for that.